docs/elements/examples/shadcn-ui.mdx

```diff
@@ -0,0 +1,502 @@
+---
+description: Learn how to compose Clerk Elements with shadcn/ui to build custom sign in and sign up flows.
+---
+
+# shadcn/ui
+
+The following example demonstrates how to compose Clerk Elements with shadcn/ui to build custom sign in and sign up flows.
+
+This example assumes you have already completed the [shadcn/ui Next.js installation guide](https://ui.shadcn.com/docs/installation/next) and have installed the [Button](https://ui.shadcn.com/docs/components/button), [Card](https://ui.shadcn.com/docs/components/card), [Input](https://ui.shadcn.com/docs/components/input), and [Label](https://ui.shadcn.com/docs/components/label) components within your project. Lastly copy the Icons component from shadcn/ui docs [here](https://github.com/shadcn-ui/ui/blob/main/apps/www/components/icons.tsx).
```

Comment:
> Lastly copy the Icons component from shadcn/ui docs [here](https://github.com/shadcn-ui/ui/blob/main/apps/www/components/icons.tsx).

This could likely benefit from a revision. Any ideas on how to best communicate manually copy pasting this file into their component directory?